src/html/Engine/syncEventList.js

```diff
@@ -132,16 +131,23 @@ function getBoxUrl(eventCellUrl, token) {
 
 // Get event details
 function getEventInfo(eventCellBoxUrl, token) {
-    var url = eventCellBoxUrl + "OData/Events?$select=__id,title,startDate,endDate,image,serviceName,serviceImage,latitude,longitude,recruiter,address,keywords";
+    var url = eventCellBoxUrl + "OData/Events";
+    var filter = "$filter=startDate%20ge%20datetimeoffset'" + moment().startOf("day").toISOString() + "'";
```

Comment:
Do you use other moment method? It is better to use moment.tz('Asia/Tokyo') because server is UTC and Client is Tokyo

Reply:
Modified to use moment.tz.